pkg/rtc/participant.go

```diff
@@ -2251,10 +2255,12 @@ func (p *ParticipantImpl) addMediaTrack(signalCid string, sdpCid string, ti *liv
 		p.pendingTracksLock.Lock()
 		if pti := p.pendingTracks[signalCid]; pti != nil {
 			p.sendTrackPublished(signalCid, pti.trackInfos[0])
+			pti.queued = false
```

Comment:
Should this check to see if `len(pti.trackInfos) == 1` before setting this to `false`, It is possible (although unlikely) that more than one is added to pending (or at least the code design is such that many requests can be added)